docs/EN/operate/build.md

```diff
@@ -1,217 +1,261 @@
 # Build and Update
 
-The Open-iAPS app can be built with two methods:
+**Jump to...**\
+Build with [GitHub](#build-open-iaps-with-github) | [Xcode](#build-open-iaps-with-xcode) | [Script](#build-open-iaps-with-script)\
+Update with [GitHub](#update-open-iaps-with-github) | [Xcode](#update-open-iaps-with-xcode) | [Source Control](#update-open-iaps-with-source-control) | [CLI](#update-open-iaps-with-cli)
 
-* [Build Open-iAPS with GitHub](#build-iaps-with-github) using a browser on any computer or tablet
-* [Build Open-iAPS with Script](#build-iaps-with-script) using a Mac with Xcode
+## Build Open-iAPS with GitHub
 
-The Open-iAPS app can be updated to the latest release or to a development branch when you choose.
+The Open-iAPS repository contains instructions for building the Open-iAPS app using a browser and GitHub. More detailed instructions are also available in Loopdocs.
 
-* [Update Open-iAPS with GitHub](#update-iaps-with-github) using a browser on any computer or tablet
-* [Update Open-iAPS with Xcode](#update-iaps-with-xcode) using a Mac with Xcode
+* [GitHub Build Instructions](https://github.com/nightscout/Open-iAPS/blob/main/fastlane/testflight.md)
```

Comment:
Should we change this to link to dev and then add a draft PR to remind us to change it back right before Oi v1.0 is released?